### ISSUE TITLE
Print date once when day changed in rooms and in open chats

### ISSUE THREAD

**Issue by c0dev0id** [profanity-im/profanity#1582]
Hello,

I started using jabber conference rooms and some of them are used to annouce irregular meetups in jitsi or big blue button. It happenes every once in a while that the last message in a room is: "We're meeting today at 18:30 here: [url]"

But then I don't know if this message was written today or yesterday or two days before.

Like here: https://codevoid.de/I/p/scr-2021-07-17_08-35-peN5ciSh.png
The last message sais: LPD orga meeting is running now: [url]. Let's assum it's around that time and I click on that link and storm into a wrong meetup cause the original meeting was a day earlier.

In IRC the same happens, but my client prints a full date line every midnight at 00:00. So I know when the day has changed and I know when the chat happened.

I'm looking for something like this in profanity.

Is there a feature like this? And if not, would this be a good idea to implement?

## Environment

Profanity, version 0.10.0
Copyright (C) 2012 - 2019 James Booth <boothj5web@gmail.com>.
Copyright (C) 2019 - 2021 Michael Vetter <jubalh@iodoru.org>.
License GPLv3+: GNU GPL version 3 or later <https://www.gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Build information:
XMPP library: libmesode
Desktop notification support: Disabled
OTR support: Enabled (libotr 4.1.1)
PGP support: Enabled (libgpgme 1.15.1)
OMEMO support: Enabled
C plugins: Enabled
Python plugins: Enabled (3.8.10)
GTK icons: Disabled

on OpenBSD-current

**Comment by mdosch** [comment]
Am 17. Juli 2021 06:43:48 UTC schrieb sdk ***@***.***>:
>Hello,
>
>I started using jabber conference rooms and some of them are used to annouce irregular meetups in jitsi or big blue button. It happenes every once in a while that the last message in a room is: "We're meeting today at 18:30 here: [url]"
>
>But then I don't know if this message was written today or yesterday or two days before.
>
>Like here: https://codevoid.de/I/p/scr-2021-07-17_08-35-peN5ciSh.png
>The last message sais: LPD orga meeting is running now: [url]. Let's assum it's around that time and I click on that link and storm into a wrong meetup cause the original meeting was a day earlier.
>
>In IRC the same happens, but my client prints a full date line every midnight at 00:00. So I know when the day has changed and I know when the chat happened.
>
>I'm looking for something like this in profanity.
>
>Is there a feature like this? And if not, would this be a good idea to implement?
>
>## Environment
>
>Profanity, version 0.10.0
>Copyright (C) 2012 - 2019 James Booth ***@***.***>.
>Copyright (C) 2019 - 2021 Michael Vetter ***@***.***>.
>License GPLv3+: GNU GPL version 3 or later <https://www.gnu.org/licenses/gpl.html>
>
>This is free software; you are free to change and redistribute it.
>There is NO WARRANTY, to the extent permitted by law.
>
>Build information:
>XMPP library: libmesode
>Desktop notification support: Disabled
>OTR support: Enabled (libotr 4.1.1)
>PGP support: Enabled (libgpgme 1.15.1)
>OMEMO support: Enabled
>C plugins: Enabled
>Python plugins: Enabled (3.8.10)
>GTK icons: Disabled
>
>on OpenBSD-current
>

You can configure profanity to always show the date as a workaround. I'm not on a computer now otherwise I'd show you an example.

**Comment by jubalh** [comment]
@c0dev0id this is a feature request, not a bugreport. When you created this bug report there was a notice not to create new feature requests. Did you read that? If yes, why did you create this feature request anyways?

That being said I'm considering to let this issue stay open since I already started with this feature quite some time ago. The branch is here: https://github.com/profanity-im/profanity/tree/feature/datechange if anybody wants to continue on that.

**Comment by c0dev0id** [comment]
> Did you read that? If yes, why did you create this feature request anyways?

Apologies, I did not. I skimmed through it, saw the usual bug-report steps "expected, actual, reproduction steps" and deleted the whole thing cause it doesn't apply to feature requests. And frankly, I was not sure if it is one or if I just can't find how to configure it.

> You can configure profanity to always show the date as a workaround.

Ah! This hint was enough. I was always looking for date settings in the room area, but it's `/time` and it's even in the example: `/time chat set "%d-%m-%y %H:%M:%S"`.

> That being said I'm considering to let this issue stay open since I already started with this feature quite some time ago. 

I think this would be nice, but as my need it fully satisfied with the workaround, I close the issue.

Sorry for the noise.
